tests/e2e/vsan_test.go

```diff
@@ -63,6 +64,10 @@ func (s *VsanTestSuite) SetUpTest(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
+	if len(s.containerList) > 0 {
+		out, err := dockercli.RemoveAllContainers(s.config.DockerHosts[0])
```

Comment:
Running test groups in parallel most likely will simply not work, given that all of these are operating off the default vmgroup and some are doing things like removing access privileges for the default vmgroup.